drinks_touch/game.py

```diff
@@ -9,15 +9,18 @@
 import sys
 import threading
 import time
+import requests
```

Comment:
is this necessary?

Reply:
Nope, good catch. Also a couple other unused imports here, will remove them.

How do you feel about adding black & flake8 with pre-commit so things like this get automatically solved?